generator_hdl.py

```diff
@@ -135,10 +138,17 @@ def writeTopModule_interface(topmodule_name, process_list, memDict, memInfoDict,
             if memInfo.isFIFO:
                 string_output_mems += writeTrackStreamRHSPorts_interface(mtypeB, memDict)
             else:
-                string_output_mems += writeMemoryRHSPorts_interface(mtypeB, memInfo)
+                  if not (("TPROJ" in mtypeB or "VMSME" in mtypeB) and args.split):
+                    string_output_mems += writeMemoryRHSPorts_interface(mtypeB, memInfo,memDict)
+              
         elif extraports:
             # Debug ports corresponding to BRAM inputs.
             string_input_mems += writeMemoryLHSPorts_interface(memList, mtypeB, extraports)            
+        if ("AS_36" in mtypeB and args.split): #for split fpga we want AS sent to second device
```

Comment:
@jasonfan393 Could we make this not depend on the bit width? Maybe something like:
```suggestion
        if (memInfo.mtype == "AllStubs" and args.split): #for split fpga we want AS sent to second device
```